lib/system_overview.rst

```diff
@@ -5,9 +5,11 @@ The System module imports several separate modules, and their documentation
 is in separate files:
 
 * `iterators <iterators.html>`_
+* `exceptions <exceptions.html>`_
 * `assertions <assertions.html>`_
 * `dollars <dollars.html>`_
 * `widestrs <widestrs.html>`_
+* `ctypes <ctypes.html>`_
```

Comment:
Need to modify `docs/kochdocs.nim` to generate documentation for "exceptions.nim" and "ctypes.nim".